### ISSUE TITLE
Jaeger merges several traces into one

### ISSUE THREAD

**Issue by clebs** [kyma-project/kyma#989]
When sending several events in quick succession Jaeger UI is merging their traces in one.
After 5 minutes, new events are put in a new trace.

### Description
The issue is due to buggy nginx/envoy behavior.
What is happening is that for even a new request, the parent traceID/spanID used is one from the previous request?

This renders a bad UX experience.

What we can do now is to disable the nginx tracing and start the trace from the `{re}-event-service`.

**Note**
Disable the tracing in following config maps
```
kyma/resources/core/charts/nginx-ingress>
$ grep -rsli tracing *
templates/zipkin-configmap.yaml
templates/controller-configmap.yaml
```
**Update** : Issue has been fixed in 0.19.0 nginx-ingress version.

### AC
- [ ] Each new request is creating a new trace
- [ ] Verify the trace in Jaeger UI conforms to the following
  `{re}-event-service --> publish --> webhook ---> push ---> lambda/microservice`

**Comment by abbi-gaurav** [comment]
### Description
The issue is due to buggy nginx/envoy behavior.
What is happening is that for even a new request, the parent traceID/spanID used is one from the previous request?

This renders a bad UX experience.

What we can do now is to disable the nginx tracing and start the trace from the `{re}-event-service`.

### AC
- [ ] Disable nginx-tracing
- [ ] Make the `{re}-event-service` configurable to either propagate trace centext headers or start a trace.
- [ ] Configure the `event-service` charts to start a new trace.
- [ ] Verify the trace in Jaeger UI conforms to the following
  `{re}-event-service --> publish --> webhook ---> push ---> lambda/microservice`

**Comment by abbi-gaurav** [comment]
Please see the PR. The issue has been fixed in 0.19.0. So initially proposed  changes are not required.

I am updating the acceptance criteria
